### DISCUSSION TITLE
Cubit vs Bloc

### DISCUSSION THREAD

**Discussion by evmartinelli** [Sameera-Perera/Flutter-TDD-Clean-Architecture-E-Commerce-App#14] [General]
I saw that in some steps you are using Cubit and in others, Bloc. Should we refactor and use only one? For example, Cubits? 

Thanks! 

Note: your project is awesome! ⭐️

**Comment by Sameera-Perera** [comment]
I don't think we should use only one type, cubits are usually used for simpler state management scenarios and Blocs are generally used for more complex scenarios where you need to handle multiple events and emit different states based on those events. In my practice I use both.

Thank you 👍
